torch/_size_docs.py

```diff
@@ -0,0 +1,38 @@
+"""Adds docstrings to torch.Size functions"""
+
+import torch._C
+from torch._C import _add_docstr as add_docstr
+
+
+def add_docstr_all(method, docstr):
+    add_docstr(getattr(torch._C.Size, method), docstr)
+
+
+add_docstr_all(
+    "numel",
+    """
+numel() -> int
+
+Returns the number of elements a :class:`torch.Tensor` with the given size would contain.
+
+More formally, for a tensor ``x = tensor.ones(10, 10)`` with size ``s = torch.Size([10, 10])``,
+``x.numel() == x.size().numel() == s.numel() == 100`` holds true.
```

Comment:
oh wow this is actually true, contrary to https://github.com/pytorch/pytorch/pull/61714#issuecomment-885404561 now. I wonder when this changed

Reply:
I think that comment was just talking about how `torch.Size.numel()` and `torch.numel()` are distinct APIs that each need to have their own docstrings.

Reply:
@soulitzer is right. The actual behavior was always like that, the comment just says those two functions are unrelated.